### ISSUE TITLE
Installation Issue

### ISSUE THREAD

**Issue by AdeleneDawner** [wizonesolutions/habitrpg-todo-sync#43]
I'm trying to install this, and not getting anywhere - it appears to work, but when I go to use it I just get an error.

Screenshot of what I'm getting:
![Imgur](http://i.imgur.com/SpgWZpK.png)

**Comment by wizonesolutions** [comment]
Yeah, I haven't updated this in forever. The code probably needs changes, in this case either for the new version of Node or just to fix some bug I'm unaware of.
